sympy/series/tests/test_limits.py

```diff
@@ -617,6 +617,10 @@ def test_issue_14393():
     assert limit((x**b - y**b)/(x**a - y**a), x, y) == b*y**(-a)*y**b/a
 
 
+def test_issue_14811():
+    assert limit(((1 + (Rational(2,3) ** (x + 1))) ** (2 ** x)) / (2 ** (Rational(4,3) ** (x - 1))), x, oo) is oo
```

Comment:
```suggestion
    assert limit(((1 + ((S(2)/3)**(x + 1)))**(2**x))/(2**((S(4)/3)**(x - 1))), x, oo) == oo
```
to reduce the length.